### PR TITLE
fix: It shouldn't be possible to create multiple group conversations 

### PR DESCRIPTION
**Pull request by makingthematrix**
fixes: https://wearezeta.atlassian.net/browse/AN-6235

### What's new in this PR?

A fix preventing the user from creating multiple conversations by clicking the "create" button many times quickly.

### Issues

It is technically possible to create many conversations with the same participants: fake 1-to-1, and real group convs. This possibilty leads to a bug when the user clicks the "Create conversation" button many times quick enough that if the app experiences a delay it is possible to create many conversations at once. 

### Solutions

To prevent that, clicks on all the buttons used for creating conversation - in the "Create group" UI, "Create guestroom", and fake 1-to-1 conversations with other team members, are now ignored while the creation of the conversation is not finished after the first click.

### Testing

The reliable way to test it requires introduction of an artificial delay in SE, in `ConversationsUiService.getOrCreateOneToOneConversation`. Without it, in most cases the first click is processed quick enough to prevent the bug. 
There are three "buttons" in the UI which are affected:
* If you're a team member, go to SearchUI and look for an *uncontacted* team member. Clicking on its name should immediately open the conversation with that person without any confirmation screen. If you click on the name many times fast, it might result in creation of many fake 1-to-1 conversations.
* SearchUI, the "Create guestroom" button. Clicking on it quickly many times may create many guestrooms.
* SearchUI, the "Create group" button. It takes the user to the Create Group UI. Provide the name of the group and choose participants. Then try to click "Done" many times quickly.

### CODE REVIEW

**Comment by makingthematrix** [comment]
build again

**Comment by marcoconti83** [comment]
Sometimes the CLA get stuck on a specific PR, but it will resume with the next ones.
Is this ready to merge? I can force merge it

**Comment by makingthematrix** [comment]
> Sometimes the CLA get stuck on a specific PR, but it will resume with the next ones.
> Is this ready to merge? I can force merge it

Yes, please merge it.

File: app/src/main/scala/com/waz/zclient/conversation/creation/CreateConversationManagerFragment.scala [hunk 1]
```diff
@@ -63,18 +64,23 @@ class CreateConversationManagerFragment extends FragmentHelper {
     case PickerPage                   => R.string.skip_button
   }
 
-  lazy val confButtonEnabled = for {
-    currentPage <- currentPage
-    name        <- ctrl.name
-    memberCount <- ctrl.users.map(_.size)
+  private val convCreationInProgress = Signal(false)
+
+  private lazy val confButtonEnabled = for {
```
Comment (CalumMcCall) [inline app/src/main/scala/com/waz/zclient/conversation/creation/CreateConversationManagerFragment.scala]: Should this not be `convButtonEnabled` instead?

**Pull request merged**